common/changes/@microsoft/tsdoc/handle-ts-built-in-directives_2025-10-30-08-48.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/tsdoc",
+      "comment": "Handle the TS built-in @jsx* directives isntead of reporting an issue when they are used.",
```

```suggestion
      "comment": "Standardize TypeScript's `@jsx`, `@jsxRuntime`, `@jsxFrag`, `@jsxImportSource` tags.",
```

Since there are only 4 of them, I think it makes sense to list them explicitly.